crawler/file_processing.py

```diff
@@ -568,20 +551,17 @@ def update_doc_from_source_plate(doc, existing_plate, skip_lab_check=False) -> N
                 "TYPE 26",
                 f"Failed assigning source plate UUIDs to samples in file {self.file_name}",
             )
-            logger.critical(
-                "Error assigning source plate UUIDs to samples in file " f"{self.file_name}: {e}"
-            )
+            logger.critical("Error assigning source plate UUIDs to samples in file " f"{self.file_name}: {e}")
             logger.exception(e)
             updated_docs = []
 
         return updated_docs
 
-    def insert_samples_from_docs_into_mongo_db(self, docs_to_insert) -> List[ObjectId]:
+    def insert_samples_from_docs_into_mongo_db(self, docs_to_insert: List[Dict[str, str]]) -> List[Any]:
```

Comment:
In short, I don't think so because a Sample type is `Dict[str, Any]` but I think the "rows" from the CSV parser will be `Dict[str, str]`. I was still getting to grips with type aliases and figuring out where best to include the aliases for readability but didn't add a complete set of aliases yet.

Reply:
:+1: